frontend/src/metabase/nav/analytics.ts

```diff
@@ -0,0 +1,11 @@
+import { trackSchemaEvent } from "metabase/lib/analytics";
+
+type LinkType = "nav" | "help";
+
+export const trackDatabasePromptBannerClicked = (link: LinkType) => {
+  trackSchemaEvent("dashboard", "1-0-1", {
+    event: "db_connection_banner_click",
+    version: "1.0.0",
```

Comment:
Other versions we've implemented are `1.0.0` which is different from [the document](https://www.notion.so/metabase/0df8245abb8f46edbede12065d65e774). But I use `1.0.0` for consistency.

https://github.com/metabase/metabase/blob/049608de5b112f90f262dc57704dc93a0513efed/frontend/src/metabase/setup/analytics.ts#L4-L27